workspacemanager/WorkspaceManagerPresenterTest.py

```diff
@@ -0,0 +1,173 @@
+from mock.mock import self
+
+from WorkspaceManagerPresenter import WorkspaceManagerPresenter
+from WorkspaceProvider import WorkspaceProvider
+from WorkspaceView import WorkspaceView
+from command import Command
+import unittest
+import mock
+from mock import call
+
+
+#ToDo askOwen, view related tests, i.e view returns 'selected_workspace' that does not exist
+#TODO handle mantid load fail inquiry
+#TODO Test constructor and make this test specific
+
+class WorkspaceManagerPresenterTest(unittest.TestCase):
+    def setUp(self):
+        self.workspace_provider = mock.create_autospec(spec=WorkspaceProvider)
+        self.view = mock.create_autospec(spec=WorkspaceView)
+
+
+
+    def test_load_one_workspace(self):
+        self.presenter = WorkspaceManagerPresenter(self.view, self.workspace_provider)
+        self.view.get_workspace_to_load_path = mock.Mock(return_value=r'C:\a\b\cde.nxs')
```

Comment:
This needs better annotation in general. For example 
```python
# Creating a view that will return a path to a nexus file on call to get_workspace_to_load_path
self.view.get_workspace_to_load_path = mock.Mock(return_value=r'C:\a\b\cde.nxs')
```
This will help the next maintainer a lot.